x-pack/solutions/search/plugins/search_connectors/server/services/index.ts

```diff
@@ -52,7 +59,14 @@ export class AgentlessConnectorsInfraService {
   public getNativeConnectors = async (): Promise<ConnectorMetadata[]> => {
     this.logger.debug(`Fetching all connectors and filtering only to native`);
     const nativeConnectors: ConnectorMetadata[] = [];
-    const allConnectors = await fetchConnectors(this.esClient);
+    const allConnectors = await fetchConnectors(
```

Comment:
Nit: At this point, perhaps we should refactor fetchConnectors to accept a single object as an argument instead of a list of arguments.
For example: `fetchConnectors({ client, includeDeleted: true })`. This feels much more readable, IMO.

Reply:
Agreed, just don't want to do it here to not make PR even bigger